puppet-lint-spaceship_operator_without_tag-check.gemspec

```diff
@@ -18,7 +18,7 @@ Gem::Specification.new do |spec|
 
   spec.required_ruby_version = '>= 2.7', '< 4'
 
-  spec.add_dependency             'puppet-lint', '>= 1.0', '< 5.0'
+  spec.add_dependency             'puppet-lint', '>= 1.0', '< 6.0'
```

Comment:
Puppet-lint 5 had some significant changes. In the other plugins we made this a major release and require ruby 3.2 / puppet-lint 5